Phoniebox/rpc/PhonieboxRpcClient.py

```diff
@@ -4,17 +4,23 @@
 class PhonieboxRpcClient:
     
     def __init__(self):
-        #self.objects = objects
         self.context = None
         
-    def connect(self,addr= None):
-        if addr == None:
-            addr = "tcp://127.0.0.1:5555"
-        self.context = zmq.Context()
+    def connect(self,addr=None,zmq_context=None):
```

Comment:
Any particular reason why you don't connect directly in the constructor?

Reply:
Yes, the clients are instantiated before the server is running in order to allow a parsing of the clients objects which need to be passed to the server.